### DISCUSSION TITLE
Selectively visualize frames

### DISCUSSION THREAD

**Discussion by richardrl** [google-deepmind/mujoco#2315] [Asking for Help]
### Intro

Hi!

I am a graduate student using Mujoco for robotics.

### My setup

Mujoco 3.2.6, Python 3.12

### My question

How can I selectively visualize frames?

With this code block, I can visualize all body frames:
```
with viewer.launch_passive(scene_model, data) as v:
    # Enable joint visualization
    v.opt.frame = mujoco.mjtFrame.mjFRAME_BODY
```
But I want to visualize only this frame, for example:
https://github.com/google-deepmind/mujoco_menagerie/blob/469893211c41d5da9c314f5ab58059fa17c8e360/franka_fr3/fr3.xml#L126C33-L126C42

Put another way, how can I only show the frames specified by a list of body name strings?

### Confirmations

- [X] I searched the [latest documentation](https://mujoco.readthedocs.io/en/latest/overview.html) thoroughly before posting.
- [X] I searched previous [Issues](https://github.com/google-deepmind/mujoco/issues) and [Discussions](https://github.com/google-deepmind/mujoco/discussions), I am certain this has not been raised before.

**Comment by yuvaltassa** [comment]
There's no way to do that.

Here's a workaround: 
1. Put a site at the body frame that you want to visualize and give it a `site_group` value that no other site has.
2. Visualize site frames, activating only that group.

Note that group values can be changed at runtime in the `mjModel`, this will not has any unexpected side-effects.
